contracts/cosmwasm-vm/cw-ibc-core/src/ics04_channel/packet/timeout.rs

```diff
@@ -104,6 +107,23 @@ impl<'a> CwIbcCoreContext<'a> {
                 },
             });
         }
+        let client = self.get_client(deps.storage, client_id_on_a)?;
```

Comment:
This does not seem to yeild the expeceted behavior?
```
   // check that timeout height or timeout timestamp has passed on the other end
    abortTransactionUnless(
      (packet.timeoutHeight > 0 && proofHeight >= packet.timeoutHeight) ||
      (packet.timeoutTimestamp > 0 && getTimestampAtHeight(connection, proofHeight) >= packet.timeoutTimestamp))
```
The behavior here seeme to be instead that both need to have expired?
Make sure the test are correct